contrib/systemd/spacenavd.service

```diff
@@ -1,12 +1,10 @@
 [Unit]
 Description=3Dconnexion Input Devices Userspace Driver
-After=syslog.target
 
 [Service]
-Type=forking
+Type=exec
 PIDFile=/var/run/spnavd.pid
-ExecStart=/usr/local/bin/spacenavd
-StandardError=syslog
```

Comment:
Here, I opt to use the systemd default, which is to put stdout and stderr lines into the systemd journal.

Now, I can do the following to see the logs for only spacenavd:
`journalctl -u spacenavd`

This is a very normal systemd pattern.